src/test/java/org/entando/kubernetes/controller/spi/client/impl/DefaultKubernetesClientForControllersTest.java

```diff
@@ -315,12 +315,31 @@ void shouldRetrieveStandardResourceGenerically() throws InterruptedException {
         step("And the same applies for Deployments, Services, Ingresses, Secrets and PersistentVolumeClaims", () -> {
             assertThat(SupportedStandardResourceKind.DEPLOYMENT.getOperation(getFabric8Client()))
                     .isInstanceOf(DeploymentOperationsImpl.class);
-            assertThat(SupportedStandardResourceKind.SERVICE.getOperation(getFabric8Client())).isInstanceOf(ServiceOperationsImpl.class);
```

Comment:
@entgigi if it's just reformat please remove it from the PR

Reply:
removed